src/component/navbar/navbar.module.scss

```diff
@@ -25,13 +26,16 @@ ul {
 }
 
 li {
-  margin: 0 1rem;  
+  margin: 0 2rem;  
 }
 
 a {
   color: black; 
   text-decoration: none;
-  font-size: 1rem;
+  font-size: 25px;
```

Comment:
sorry @Kendiya242 , I didn't see this specific change that altered the navbar styling and you didn't specify which lines of code you changed hence I was under the assumption that there were not any changes but minor. Considering that most of the files just had spaces I did not interrogate closer.

Reply:
its ok next time I will state which specific changes I made on a next PR.